test/specs/components/Popup/Popup-test.tsx

```diff
@@ -56,4 +61,45 @@ describe('Popup', () => {
     testPopupPositionInRtl({ position: 'after', align: 'center', expectedPlacement: 'left' })
     testPopupPositionInRtl({ position: 'after', align: 'bottom', expectedPlacement: 'left-end' })
   })
+
+  describe('Popup offset transformed correctly in RTL', () => {
+    it("applies transform only for 'above' and 'below' postioning", () => {
+      const originalOffsetValue = '100%'
+
+      expect(applyRtlToOffset(originalOffsetValue, 'above')).not.toBe(originalOffsetValue)
+      expect(applyRtlToOffset(originalOffsetValue, 'below')).not.toBe(originalOffsetValue)
+
+      expect(applyRtlToOffset(originalOffsetValue, 'before')).toBe(originalOffsetValue)
+      expect(applyRtlToOffset(originalOffsetValue, 'after')).toBe(originalOffsetValue)
+    })
+
+    const expectOffsetTransformResult = (originalOffset, resultOffset) => {
+      expect(applyRtlToOffset(originalOffset, 'above')).toBe(resultOffset)
+    }
+
+    it('flips sign of simple expressions', () => {
+      expectOffsetTransformResult('100%', '-100%')
+      expectOffsetTransformResult('  2000%p ', '-2000%p')
+      expectOffsetTransformResult('100  ', '-100')
+      expectOffsetTransformResult(' - 200vh', '200vh')
```

Comment:
You can use `jest.each` there: https://jestjs.io/docs/en/api#testeachtable-name-fn-timeout